### DISCUSSION TITLE
Exporting papers from ASReview and converting file from xlsx to xml

### DISCUSSION THREAD

**Discussion by cchiara1** [asreview/asreview#1658] [Q&A]
Dear ASReview community,

Once the initial screening is completed on ASReview, a person should export the relevant papers to proceed further. In order to export, I need to convert the file from excel format to xml format. However, I am not able to create and save a XML file. Do you have any suggestions to solve this issue?

Thank you in advance!

**Comment by FelixWdm** [comment]
Hi @cchiara1 , once finished you can export the complete project, for reference. And export the relevant papers in csv or RIS format. If you select RIS, you can import all, or just the relevant ones in almost any reference manager and continue from there. For what application do you need the XML?

![afbeelding](https://github.com/asreview/asreview/assets/49488769/133077ac-7fa8-4fbc-aa49-1a4478d8668a)
